docs/terraformoptions.adoc

```diff
@@ -456,22 +456,22 @@ admission_controller_options = {
 |v1.16.8
 
 |`node_pools`
-|The number, shape and quantities per subnets of node pools to create. Each key and tuple pair corresponds to 1 node pool. The first parameter in the tuple sets the shape of the worker node, the 2nd parameter sets the size of the node pool and the 3rd parameter the boot volume size in GB. An empty value ("") for the boot volume size will default the boot volume size to 50GB. A minimum of 1 worker worker nodes per node pool will be created.  Refer to {uri-topology}[topology] for more thorough examples.
+|The number, shape and quantities per subnets of node pools to create. Each key and tuple pair corresponds to 1 node pool. The first parameter in the tuple sets the shape of the worker node,the 2nd parameter sets the number of OCPU for flex shape compute and minimum one OCPU will be used. Its set to null for standard shapes, the 3rd parameter sets the size of the node pool and the 4th parameter the boot volume size in GB. An empty value ("") for the boot volume size will default the boot volume size to 50GB. A minimum of 1 worker worker nodes per node pool will be created.  Refer to {uri-topology}[topology] for more thorough examples.
```

Comment:
We don't use quantities per subnets for node pools anymore. We use cluster size

Reply:
don't set to null for standard shapes. just ignore it altogether. see if you can add dynamic before the node_shape_config.

Reply:
Also add that the ocpus is for each worker node.

Reply:
> don't set to null for standard shapes. just ignore it altogether. see if you can add dynamic before the node_shape_config.

https://docs.cloud.oracle.com/en-us/iaas/api/#/en/iaas/20160918/datatypes/ShapeOcpuOptions
ocpus is a number type.Its better to pass null as we are assigning the same in the code as well.